modules/app-contentstudio/src/main/resources/assets/styles/variables.less

```diff
@@ -0,0 +1 @@
+@mobile-panel-width: 720px;
```

Comment:
I think this setting belongs [here](https://github.com/enonic/enonic-npm-modules/tree/master/packages/enonic-admin-artifacts/src/size.less). Read [manual](https://github.com/enonic/enonic-npm-modules/blob/master/PUBLISH.md) on how to update the npm package or ask @edloidas for details.

Reply:
@alansemenov  I had disscussed it with @edloidas before it was commited and we think there is no reason to move contentstudio's styles to commons